src/bindgen/index.md

```diff
@@ -20,6 +20,8 @@ use Rust in a C/C++ codebase in the first place!
 
 ## Getting Set Up
 
+> **Note:** Bindgen **requires** clang. Installation instructions can be found [here](https://github.com/rust-lang-nursery/rust-bindgen/blob/master/book/src/requirements.md#installing-clang-39)
```

Comment:
No, not yet.  Here's the diff of the errors between the two dev consoles (io version and localhost)
```
< "WARNING: cdn.mathjax.org has been retired. Check https://www.mathjax.org/cdn-shutting-down/ for migration tips."  MathJax.js:32:5
< Expected ‘none’, URL, or filter function but found ‘alpha(’.  Error in parsing value for ‘filter’.  Declaration dropped.  book.css:192:10
< Error in parsing value for ‘display’.  Declaration dropped.  book.css:217:11
< Error in parsing value for ‘display’.  Declaration dropped.  book.css:218:11
---
> Expected ‘none’, URL, or filter function but found ‘alpha(’.  Error in parsing value for ‘filter’.  Declaration dropped.  book.css:196:10
> Error in parsing value for ‘display’.  Declaration dropped.  book.css:221:11
> Error in parsing value for ‘display’.  Declaration dropped.  book.css:222:11
8a8,10
> Error in parsing value for ‘white-space’.  Declaration dropped.  book.css:815:17
> Error in parsing value for ‘white-space’.  Declaration dropped.  book.css:816:17
> Error in parsing value for ‘white-space’.  Declaration dropped.  book.css:817:17
15d16
< "WARNING: cdn.mathjax.org has been retired. Check https://www.mathjax.org/cdn-shutting-down/ for migration tips."  MathJax.js:32:5
```